components/feed/FeedItem.tsx

```diff
@@ -14,11 +15,15 @@ const FeedItem = ({
   <div>
     <div className="h-fit">
       <div className="flex flex-row justify-between items-center">
-        {showLink ? (
-          <UsernameLink username={username} />
-        ) : (
-          <div>{username}</div>
-        )}
+        <div className="flex flex-row">
+          {/* @ts-expect-error Server Component */}
+          <ProfileImg username={username} />
+          {showLink ? (
+            <UsernameLink username={username} />
+          ) : (
+            <div>{username}</div>
```

Comment:
```suggestion
            <p>{username}</p>
```

I think this should be a p, try it and see if it breaks the design